kafka-ui-react-app/src/components/Topics/Topic/Schema/Schema.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import useAppParams from 'lib/hooks/useAppParams';
+import { RouteParamsClusterTopic } from 'lib/paths';
+import { useAppDispatch, useAppSelector } from 'lib/hooks/redux';
+import { resetLoaderById } from 'redux/reducers/loader/loaderSlice';
+import {
+  fetchLatestSchema,
+  getSchemaLatest,
+  getAreSchemaLatestFulfilled,
+  getAreSchemaLatestRejected,
+  SCHEMA_LATEST_FETCH_ACTION,
+} from 'redux/reducers/schemas/schemasSlice';
+import LatestVersionItem from '../../../Schemas/Details/LatestVersion/LatestVersionItem';
```

Comment:
Sure, no worries let me fix that, when I'm home from vacation at my computer with this code 😊

Reply:
@Mgrdich let's chill with the review, we might not need it, as I asked to raise a PR for an example, since we wanted to achieve something similar but not that similar :))